benchmark/conversions/conversions.cpp

```diff
@@ -156,8 +162,7 @@ int main(int argc, char *argv[])
             try {
                 auto matrix_from =
                     share(formats::matrix_factory.at(format_from)(exec, data));
-                for (const auto &format : formats::matrix_factory) {
-                    const auto format_to = std::get<0>(format);
+                for (const auto &format_to : formats) {
```

Comment:
does it get the pair (key, value)? what is the string of format_from and format_to

Reply:
before, we tested the conversions from the input formats to all available formats, which is pretty much overkill since most of them are duplicate, and only differ in their strategy settings. `formats` is the list of formats from `FLAGS_formats`, so instead of doing FLAGS_formats x all_formats, we now just do FLAGS_formats x FLAGS_formats conversions.

Reply:
I see. the formats and namespaces have the same name such that I am confused with